File: app/src/main/java/com/fuchsia/shotokanwkf/katamodel.java
package com.fuchsia.shotokanwkf;

public class katamodel {

    String Name,Pic,URL;
    katamodel()
    {

    }
    public katamodel(String Name, String Pic, String URL) {
        this.Name = Name;
        this.Pic= Pic;
        this.URL = URL;
    }

    public String getName() {
        return Name;
    }

    public void setName(String Name) {
        this.Name = Name;
    }

    public String getPic() {
        return Pic;
    }

    public void setPic(String Pic) {
        this.Pic = Pic;
    }

    public String getURL() {
        return URL;
    }

    public void setURL(String URL) {
        this.URL = URL;
    }
}

File: app/src/main/java/com/fuchsia/shotokanwkf/model/Katamodel.kt
package com.fuchsia.shotokanwkf.model

class katamodel {
    var name: String? = null
    var pic: String? = null
    var url: String? = null

    internal constructor() {}
    constructor(Name: String?, Pic: String?, URL: String?) {
        name = Name
        pic = Pic
        url = URL
    }
}